From 137175fb5a4dbee1df811977a43f76c89685a4de Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Fri, 28 Jul 2017 11:48:23 +0800
Subject: [PATCH 2256/5242] MLK-16062-5: dts: i.MX8QM arm2 default support 4
 mipi sensors

commit  d7f4ae6c7d8ca99bf9617251ce0a0aeb59ad03bc from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX8QM can support 4 camera sensors or 8 camera sensors.

For eight sensors case, it must need all eight sensors on board,
and for four sensors case, it must need all four sensors on board,
otherwise all mipi sensor can't work.

Four mipi sensors case is more popular than eight sensors.
so change to default support four sensors.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index aed76bb..ca9e8d5 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -554,7 +554,7 @@
 	#size-cells = <0>;
 	pinctrl-names = "default";
 	clock-frequency = <100000>;
-	status = "okay";
+	status = "disabled";
 
 	max9286_mipi@6A	 {
 		compatible = "maxim,max9286_mipi";
@@ -728,7 +728,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 	virtual-channel;
-	status = "okay";
+	status = "disabled";
 
 	/* Camera 0 MIPI CSI-2 (CSIS1) */
 	port@1 {
-- 
1.7.9.5

